packages/terra-search-field/src/SearchField.scss

```diff
@@ -4,24 +4,26 @@
 
     .input {
       appearance: none;
+      border-bottom-left-radius: var(--terra-search-field-border-bottom-left-radius, 0.25em);
       border-bottom-right-radius: 0;
       border-right: 0;
+      border-top-left-radius: var(--terra-search-field-border-top-left-radius, 0.25em);
       border-top-right-radius: 0;
       flex: 1 1 auto;
-      margin: 0;
+      margin: var(--terra-search-field-input-margin, 0);
```

Comment:
We've had issues using theming variables with margin/padding shorthand and bidirectional styles (LTR/RTL). Let's split this out to long-hand style:

```css
margin-bottom: var(--terra-search-field-input-margin-bottom, 0);
margin-left: var(--terra-search-field-input-margin-left, 0);
margin-right: var(--terra-search-field-input-margin-right, 0);
margin-top: var(--terra-search-field-input-margin-top, 0);
```